src/LaborMarket/LaborMarket.sol

```diff
@@ -231,38 +162,46 @@ contract LaborMarket is
             "LaborMarket::signal: Already signaled."
         );
 
-        // Lock reputation here
-        uint256 signalAmt = 1;
+        uint256 signalStake = reputationModule.getSignalStake(address(this));
+
+        _lockReputation(
+            msg.sender,
+            signalStake
+        );
 
         submissionSignals[requestId][msg.sender] = true;
 
         unchecked {
             ++signalCount[requestId];
         }
 
-        emit RequestSignal(msg.sender, requestId, signalAmt);
+        emit RequestSignal(msg.sender, requestId, signalStake);
     }
 
     /**
      * @notice Signals interest in reviewing a submission.
-     * @param submissionId The id of the service providers submission.
+     * @param requestId The id of the service providers submission.
      */
-    function signalReview(uint256 submissionId) external onlyMaintainer {
+    function signalReview(uint256 requestId) external onlyMaintainer {
         require(
-            submissionId <= serviceSubmissionId,
-            "LaborMarket::signalReview: Submission does not exist."
+            requestId <= serviceRequestId,
+            "LaborMarket::signalReview: Request does not exist."
         );
         require(
-            !reviewSignals[submissionId][msg.sender],
+            !reviewSignals[requestId][msg.sender],
             "LaborMarket::signalReview: Already signaled."
         );
 
-        // Lock reputation here
-        uint256 signalAmt = 1;
+        uint256 signalStake = reputationModule.getSignalStake(address(this));
 
-        reviewSignals[submissionId][msg.sender] = true;
+        _lockReputation(
+            msg.sender,
+            signalStake
+        );
 
-        emit ReviewSignal(msg.sender, submissionId, signalAmt);
+        reviewSignals[requestId][msg.sender] = true;
```

Comment:
Review signaling is no longer a flat single level of interest. Has been updated in this [PR](https://github.com/FlipsideCrypto/labor-market/pull/3) and will need to be addressed here as well, but to merge into the existing structure, this looks nice :) Just needs an update for the future!